tests/migrate/test_plural.py

```diff
@@ -142,5 +143,81 @@ def test_plural_replace(self):
         )
 
 
+@unittest.skipUnless(PropertiesParser, 'compare-locales required')
+class TestPluralOneCategory(MockContext):
+    # Plural categories corresponding to Turkish (tr).
+    plural_categories = ('other',)
+
+    def setUp(self):
+        self.strings = parse(PropertiesParser, '''
+            deleteAll=#1 indirme silinsin mi?
+        ''')
+
+        self.message = FTL.Message(
+            FTL.Identifier('delete-all'),
+            value=PLURALS(
+                'test.properties',
+                'deleteAll',
+                EXTERNAL_ARGUMENT('num'),
+                lambda text: REPLACE_IN_TEXT(
+                    text,
+                    {
+                        '#1': EXTERNAL_ARGUMENT('num')
+                    }
+                )
+            )
+        )
+
+    def test_plural_without_select_expression(self):
+        self.assertEqual(
+            evaluate(self, self.message).to_json(),
+            ftl_message_to_json('''
+                delete-all = { $num } indirme silinsin mi?
+            ''')
+        )
+
+
+@unittest.skipUnless(PropertiesParser, 'compare-locales required')
+class TestPluralManyCategoriesWithReplace(MockContext):
```

Comment:
Can you rename the test? Right now, this is test_plural.TestPluralManyCategoriesWithReplace.test_plural.

One plural is enough, how about TestManyCategories.test_with_replace()?

That said, this test sounds more like a test for the upcoming patch about strictly matching plural forms between toolkit and cldr?